### PR TITLE
Guild Utility Functions

### PR DESCRIPTION
**Pull request by skippi**
This adds in two functions for the guild module:

* `image_url/2`: For retrieving a guild's icon URL.
* `splash_url/2`: For retrieving a guild's splash URL.

### CODE REVIEW

File: lib/nostrum/struct/guild.ex [hunk 1]
```diff
@@ -287,6 +287,60 @@ defmodule Nostrum.Struct.Guild do
...
+  iex> Nostrum.Struct.Guild.icon_url(guild)
+  nil
+  ```
+  """
+  @spec icon_url(t, String.t()) :: String.t() | nil
+  def icon_url(guild, image_format \\ "webp")
+  def icon_url(%__MODULE__{icon: nil}, _), do: nil
+
+  def icon_url(%__MODULE__{icon: icon, id: id}, image_format),
+    do: "https://cdn.discordapp.com/icons/#{id}/#{icon}.#{image_format}"
```
Comment (Kraigie) [inline lib/nostrum/struct/guild.ex]: I think we should move these URLs to the constants file

**Comment by skippi** [comment]
Putting on hold until #82 is complete.

File: lib/nostrum/struct/guild.ex (hunk 1 above)
Comment (skippi) [reply lib/nostrum/struct/guild.ex]: Addressed in af9af94.

**Comment by skippi** [comment]
Ready for more code review.

**Comment by Kraigie** [comment]
Thanks!

**Pull request merged**